packages/camera/camera_web/lib/src/camera_service.dart

```diff
@@ -307,20 +307,23 @@ class CameraService {
     return const Size(320, 240);
   }
 
+  static const int _K = 1000;
+  static const int _M = _K * _K;
```

Comment:
I don't mind too much, but the Effective Dart guide [discourages abbreviations](https://dart.dev/effective-dart/design#avoid-abbreviations) and if used, [capitalized properly](https://dart.dev/effective-dart/style#do-name-other-identifiers-using-lowercamelcase).

I'd suggest this to be:

```suggestion
  static const int _kiloBits = 1000;
  static const int _megaBits = _kiloBits * _kiloBits;
```

(Or similar)